composer.json

```diff
@@ -9,7 +9,7 @@
   },
   "require-dev": {
     "squizlabs/php_codesniffer": "*",
-    "wp-coding-standards/wpcs": "0.10.0",
+    "wp-coding-standards/wpcs": "2.3.0",
```

Comment:
Update wp-coding-standards to ensure we are complying with the latest code standards.